tools/api-client/python/lib/bmutils.py

```diff
@@ -98,10 +100,40 @@ def wrap_create_game(self, pbutton, obutton='', opponent=''):
     return retval.data
 
   def wrap_load_game_data(self, game):
-    retval = self.load_game_data(game)
-    if not retval.status == 'ok':
-      raise ValueError, "Failed to call loadGameData, got: " + retval
-    data = retval.data
+    # if we're using a cache directory
+    if self.cachedir:
+      # if the cache directory doesn't exist, create it
+      if not os.path.isdir(self.cachedir):
+        os.makedirs(self.cachedir)
+      # set the path to the cache file for this game
+      cachefile = self.cachedir + "/{0:010d}.json".format(game)
```

Comment:
Hmm.  This used to work if you passed game as either an int or a string, now it blows up if you pass a string.  Is that okay?  Maybe it's okay.  It blows up my scripts, but i can modify them.  Let me check to see if it blows up anything which we have committed to the repo.